### PR TITLE
[CosmosDB] Prepare release 1.2.0

### PR DESCRIPTION
**Pull request by fengzhou-msft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs [hunk 1]
```diff
@@ -2898,12 +2898,18 @@ public partial class MongoDBPrivilege
...
         public MongoDBPrivilege() { }
         public System.Collections.Generic.IList<string> Actions { get { throw null; } }
-        public Azure.ResourceManager.CosmosDB.Models.PrivilegeResource Resource { get { throw null; } set { } }
+        public Azure.ResourceManager.CosmosDB.Models.MongoDBPrivilegeResourceInfo Resource { get { throw null; } set { } }
+    }
+    public partial class MongoDBPrivilegeResourceInfo
+    {
+        public MongoDBPrivilegeResourceInfo() { }
+        public string Collection { get { throw null; } set { } }
+        public string DB { get { throw null; } set { } }
```
Comment (Yao725) [inline sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs:2907]: This property name is too generic, can we rename it?

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs [hunk 2]
```diff
@@ -2898,12 +2898,18 @@ public partial class MongoDBPrivilege
...
     }
     public partial class MongoDBRole
     {
         public MongoDBRole() { }
-        public string Db { get { throw null; } set { } }
+        public string DB { get { throw null; } set { } }
```
Comment (Yao725) [inline sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs:2912]: Same as above.

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs (hunk 1 above)
Comment (fengzhou-msft) [reply sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs]: Updated to `DBName`

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs (hunk 2 above)
Comment (fengzhou-msft) [reply sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs]: Updated to `DBName`

**Review by Yao725 — APPROVED** [review]
LGTM

**Pull request merged**